pyproject.toml

```diff
@@ -45,8 +45,9 @@ sentencepiece = { version = "^0.1.97", optional = true }
 tensorboard = { version = "^2.11.2", optional = true }
 sklearn = { version = "^0.0.post1", optional = true }
 scikit-learn = { version = "^1.2.1", optional = true }
-pytorch-lightning = { version = "^1.8.3", optional = true }
+pytorch-lightning = { extras = ["extra"], version = "^1.8.3", optional = true }
 timm = { version = "^0.6.11", optional = true }
+jsonargparse = {extras = ["signatures"], version = "^4.20.0"}
```

Comment:
It should work without this dependency since you don't use it directly and it is already included in `pytorch-lightning[extra]`. After removing it, don't forget to run `poetry update` again to update `poetry.lock` (although dependencies shouldn't really change I think the lock file also maybe specifies if it's a direct or indirect dependency).

Reply:
Btw if it doesn't work after removing `jsonargparse`, I am thinking that maybe you should use `"pytorch-lightning[extra]"` instead of just `"pytorch-lightning"` in the `baselines = [...]` list a few lines below.

Reply:
So I removed the line from `pyproject.toml`, ran `poetry update` and checked that training works.

Adding `[extra]` gives me syntax errors.

Reply:
Great, I wasn't sure if it's supposed to be there or not, as long as it works it's fine.